scripts/sysman/EXT_RasState.rst

```diff
@@ -0,0 +1,28 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+    s=tags['$s']
+    S=s.upper()
+%>
+:orphan:
+
+.. _ZES_extension_ras_state:
+
+========================
+ Ras Get State Extension
+========================
+
+API
+----
+
+* Functions
+
+    * ${s}RasGetStateExt
```

Comment:
do we want this as an standard or experimental extension?

Reply:
I prefer standard for this case as we want users to use this interface reliably going forward to gather Ras state struct for error categories. it's not intended to be experimental, but more definitive/stable mechanism to users